internal/sync/lifecycle_test.go

```diff
@@ -155,7 +216,7 @@ func TestLifecycleOnce(t *testing.T) {
 						},
 						StartAction{
 							Wait:          40 * time.Millisecond,
-							ExpectedState: Stopped,
+							ExpectedState: Running,
```

Comment:
iiiiiiinteresting, because the StartAction short-circuits the actual execution it could be running at this point, makes sense.